app/views/trainees/training_details/edit.html.erb

```diff
@@ -7,7 +7,11 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <%= register_form_with(model: @training_details_form, url: trainee_training_details_path(@trainee), method: :put, local: true) do |f| %>
-      <%= f.govuk_error_summary %>
+      <% if f.object.duplicate_error? %>
+        <%= render "duplicate_error", form: f %>
```

Comment:
Is it possible to do this with the presenter approach @ralph-hawkins linked to rather than having to duplicate most of the error summary markup?

Reply:
@gpeng I looked at that solution but unfortunately, the presenter only allows you to change the text of the list item text only. Without presenter would have to define the form builder.
This was the simpler solution.